Gruntfile.js

```diff
@@ -12,16 +12,16 @@ module.exports = function(grunt) {
         },
 
         browserify: {
-            dist: {
-                src: ['node_modules/jquery/dist/jquery.min.js', 'node_modules/leaflet/dist/leaflet.js', 'src/leafletEnvironmentalLayers.js', 'src/util/*.js'],
-                dest: 'dist/LeafletEnvironmentalLayers.js'
-            },
-            babel: {
-              files: {
-                "dist/util/layersBrowser.js": "dist/util/layersBrowser_babel.js"
-              }
+          dist: {
+              src: ['node_modules/leaflet/dist/leaflet.js', 'src/leafletEnvironmentalLayers.js', 'src/util/*.js'],
```

Comment:
@crisner Do you know if we have to have jquery in the dist file? I removed it to make the file size smaller and everything seems to work fine, but I want to make sure there's not some other reason it's in there.

Reply:
Not sure about that myself. @sagarpreet-chadha, do you know if it is required to include jquery here?